camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -1,358 +1,471 @@
+                    id="task-form-tab"
+                    data-bs-toggle="tab"
+                    data-bs-target="#taskForm"
+                    type="button"
+                    role="tab"
+                    aria-controls="form"
+                    aria-selected="true"
+                  >Form</button>
+                </li>
+                <li
+                  class="nav-item"
+                  role="presentation"
+                >
+                  <button
+                    class="nav-link"
+                    id="task-history-tab"
+                    data-bs-toggle="tab"
+                    data-bs-target="#taskHistory"
+                    type="button"
+                    role="tab"
+                    aria-controls="history"
+                    aria-selected="false"
+                  >History</button>
+                </li>
+                <li
+                  class="nav-item"
+                  role="presentation"
+                >
+                  <button
+                    class="nav-link"
+                    id="task-diagram-tab"
+                    data-bs-toggle="tab"
+                    data-bs-target="#diagramContainer"
+                    type="button"
+                    role="tab"
+                    aria-controls="diagram"
+                    aria-selected="false"
+                    @click="getDiagramDetails"
+                  >Diagram</button>
+                </li>
+              </ul>
+              <div class="tab-content py-3 task-tab-content">
+                <!-- Form tab content -->
+                <div
+                  class="tab-pane fade show active form-tab-content"
+                  id="taskForm"
+                  role="tabpanel"
+                  aria-labelledby="form-tab"
+                  :class="{'disabled': task.assignee !== userName}"
+                >
+                  <div v-if="task.assignee === userName">
+                    <transition name="fade">
+                      <FormEdit
+                        :formioUrl="formioUrl"
+                        @onformsubmit="onFormSubmitCallback"
+                        @oncustomevent="oncustomEventCallback"
+                      />
+                    </transition>
                   </div>
-                </b-tab>
-                <b-tab title="History">
+                  <div v-else>
+                    <FormView :formioUrl="formioUrl" />
+                  </div>
+                </div>
+                <!-- Form tab content end -->
+                <div
+                  class="tab-pane fade"
+                  id="taskHistory"
+                  role="tabpanel"
+                  aria-labelledby="history-tab"
+                >
                   <TaskHistory
                     :taskHistoryList="taskHistoryList"
                     :applicationId="task.applicationId"
                   />
-                </b-tab>
-                <b-tab
-                  class="cft-diagram-container"
```

Comment:
I am seeing that some of the class starting with `cft` is removed. it may impact some of the existing clients like serviceBC, because, these classes are used by clients for styling purposes. @vinod-aot @krishnan-aot once this is merged, this can be tested on the existing client-side